apps/src/lib/script-editor/ScriptEditor.jsx

```diff
@@ -567,12 +567,16 @@ const VisibleInTeacherDashboard = props => (
   </label>
 );
 VisibleInTeacherDashboard.propTypes = {
+  paramName: PropTypes.string,
   checked: PropTypes.bool,
   disabled: PropTypes.bool,
   onChange: PropTypes.func.isRequired
 };
+VisibleInTeacherDashboard.defaultProps = {
+  paramName: 'visible_to_teachers'
```

Comment:
Instead of making this a default param it might be better to add it to the i18n strings list so that it will be set up for future translation.   You can add a key:value pair here: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/i18n/common/en_us.json and then use it like this example: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/code-studio/components/BubbleChoice.jsx#L135